src/makeRobot.js

```diff
@@ -38,7 +38,63 @@
  * @return {object}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const Robot = {
```

Comment:
```suggestion
  const robot = {
```
З великої літери пишуться назви класів